graviti/portex/template.py

```diff
@@ -90,6 +90,21 @@ def get_keys(self, type_name: Optional[str] = None) -> List[Tuple[str, ...]]:
         return self.to_builtin().get_keys(type_name)
 
 
+class PortexExternalRecordType(PortexExternalType):  # pylint: disable=abstract-method
```

Comment:
can we make a class, maybe `RecordBase` as the base class for both `record` and other record-like types?